### PR TITLE
MPC examples update

### PR DESCRIPTION
**Pull request by ihsraham**
* Typo changes in folder
* Added readmes to all MPC examples
* Added multi account feature
* Index gets stored in the local storage and is retrieved/updated as per discretion.
* Gets back the index the app corekitInstance had before add/deleting/updating any factor
* Resets the stored index to 0 on resetting the account

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -20,5 +20,25 @@
...
+        "statusBar.foreground": "#15202b",
+        "statusBarItem.hoverBackground": "#007fff",
+        "statusBarItem.remoteBackground": "#3399ff",
+        "statusBarItem.remoteForeground": "#15202b",
+        "titleBar.activeBackground": "#0066cc",
+        "titleBar.activeForeground": "#e7e7e7",
+        "titleBar.inactiveBackground": "#0066cc99",
+        "titleBar.inactiveForeground": "#e7e7e799"
+    },
+    "peacock.color": "#007fff"
```
Comment (matthiasgeihs) [inline .vscode/settings.json:43]: Revert this change. We don't want to force developers to a specific color scheme. Actually, it's usually best to not commit any IDE specific settings. Is there a reason why we include this file at all?

Comment (ihsraham) [reply .vscode/settings.json]: Resolved
